pydeps/mf27.py

```diff
@@ -59,13 +60,13 @@ def load_module(self, fqname, fp, pathname, file_info):
                 dont_inherit=True  # [pydeps] don't inherit future statements from current environment
             )
 
-        elif kind == imp.PY_COMPILED:
-            # a .pyc file is a binary file containing only thee things:
+        elif kind == modulefinder._PY_COMPILED:
+            # a .pyc file is a binary file containing only three things:
             #  1. a four-byte magic number
             #  2. a four byte modification timestamp, and
             #  3. a Marshalled code object
             # from: https://nedbatchelder.com/blog/200804/the_structure_of_pyc_files.html
-            if fp.read(4) != imp.get_magic():
+            if fp.read(4) != PYC_MAGIC_NUMBER:
```

Comment:
Not sure at all about this.

Reply:
>  imp.get_magic()
> 
>     Return the magic string value used to recognize byte-compiled code files (.pyc files). (This value may be different for each Python version.)

so we can't just define it... but it's available from `importlib.util.MAGIC_NUMBER` (since py3.4).

Reply:
Ah, the last part was missing from `help(imp.get_magic)`. I'll use `importlib.util.MAGIC_NUMBER`, thanks.

Reply:
Use `importlib.util.MAGIC_NUMBER`.